echo-notifications/src/test/groovy/com/netflix/spinnaker/echo/notification/ManualJudgmentTemplateTest.groovy

```diff
@@ -122,4 +122,37 @@ class ManualJudgmentTemplateTest extends Specification {
         return notification
     }
 
+  void "should not word wrap (used for slack)"() {
+    given:
+    Notification notif = buildFullNotification()
+    notif.additionalContext["instructions"]= '''\
+      This is an example message from spinnaker that's more than 80 characters to make sure that we won't wrap it. Note the extra whitespace at the end here:
+      <p />
+      ```
+      This test was added was because code blocks in Slack would word wrap at 80 characters and inject newlines in codeblocks. Note the extra whitespaces after each original newline.
+      ```
+      '''.stripIndent()
+
+    when:
+    def rendered = notificationTemplateEngine.build(notif, NotificationTemplateEngine.Type.BODY)
+
+    then:
+    rendered == expected
+
+    where:
+
+    // Note, there's a whitespace after each new line of the `instructions` because of `Jsoup.parse(...)`.
+    // This is because the field is HTML parsed to plaintext. Which also can contain Slack's psudo-markdown flavor.
```

Comment:
Note, I did find an existing issue with `Jsoup.parse(...)`. It adds whitespaces after `\n` breaks... which is pretty annoying, but let's not deal with that in this PR.